bdb-ip.md

```diff
@@ -0,0 +1,437 @@
+# Improvement proposals for HTTP API
+
+
+Currently, there is two milestones grouping issues for upcoming HTTP API
+endpoints:
+
+- [Changes that break the existing HTTP API of BDB
+  v0.11](https://github.com/bigchaindb/bigchaindb/milestone/11)
+- [Changes that are additions to the exiting HTTP API of BDB
```

Comment:
issue #1169 proposes a change on the data that is returned by the endpoint. Should this be considered a breaking change?